test/asana/helpers/test_extract_task_fields_from_pull_request.py

```diff
@@ -424,6 +453,64 @@ def test_completed_is_false_if_pr_is_closed_and_was_approved_before_merging_but_
         )
         self.assertEqual(False, task_fields["completed"])
 
+    @patch(
+        "src.github.logic.SGTM_FEATURE__FOLLOWUP_REVIEW_GITHUB_USERS", [followup_bot]
```

Comment:
I'm kind of confused how this is working - since `SGTM_FEATURE__FOLLOWUP_REVIEW_GITHUB_USERS` should be a set of integers, but `[followup_bot]` is an array of `User`s, how does this code work?

Reply:
You know, that's a great question...

Reply:
@mattbryant-asana  check out my latest commit -- the one test was using `merged` rather than `merged_at` (I think hitting into the new condition you added where there's no `merged_at` value)

the second was passing because it didn't really matter what the value of the followup user was since it's more of a regression test to make sure no functionality is introduced to prevent completing the task if a human already reviewed before the followup user